### ISSUE TITLE
Want the string batch to be copied laterally

### ISSUE THREAD

**Issue by Thewisem** [TCreopargh/Text-Converter-Android#1]
I want the word to be copied in parallel way instead of a continous string.

**Comment by TCreopargh** [comment]
In string replacement enable regex and replace \w+ with $val $val

**Comment by Thewisem** [comment]
Ummmm...

**Comment by Thewisem** [comment]
I was talking about the Toolbox string batch copy

**Comment by Thewisem** [comment]
I can't use regex there

**Comment by Thewisem** [comment]
For your method I have to do this for each time I wanted to copy. So If I wanted to copy for a thousand times. I have to write $val a thousand times

**Comment by TCreopargh** [comment]
Do it in the replace section
If you want to copy it for a large number of times just use the batch copy to generate a large quantity of "$val "s

**Comment by Thewisem** [comment]
It will generate in a continous stream rather than parallel

**Comment by Thewisem** [comment]
Instead of like this
Hellohellohello
I want it like this
Hello
Hello
Hello

**Comment by TCreopargh** [comment]
Just add a "\n" between the "$val"s

**Comment by Thewisem** [comment]
![Screenshot_20210411-140610](https://user-images.githubusercontent.com/74511929/114301840-19ff4680-9acf-11eb-83a8-9ea174582594.jpg)

**Comment by Thewisem** [comment]
Still the same thing

**Comment by TCreopargh** [comment]
Not the \n, instead just hit enter so the replace field looks like this
$val
$val

**Comment by Thewisem** [comment]
But that means that if I wanted it in large quantities, I have to do that a lot of times

**Comment by TCreopargh** [comment]
No, you can also input a new line in toolbox to generate a pattern like that

**Comment by Thewisem** [comment]
Oh yeah about that

**Comment by Thewisem** [comment]
Already tried that

**Comment by Thewisem** [comment]
![Screenshot_20210411-141557](https://user-images.githubusercontent.com/74511929/114302151-7adb4e80-9ad0-11eb-969f-a215d45c857f.jpg)

**Comment by Thewisem** [comment]
It creates a second word at the middle words

**Comment by Thewisem** [comment]
Had already done this before asking

**Comment by TCreopargh** [comment]
No, just 
```
hello 
[New line]
```

**Comment by Thewisem** [comment]
Oh thanks, it works. Thanks a lot. :)

**Comment by Thewisem** [comment]
Now it it time to spam something which my friends will go crazy. Lol

**Comment by Thewisem** [comment]
Anyways thanks for taking your time. I was half expecting the developer to not be there
